### ISSUE TITLE
Example for generating qbSchema from JSON schema

### ISSUE THREAD

**Issue by slouc** [qb-project/qbproject#56]
You say "qbSchema can also be generated by incoming JSON schema" but you only provide the example which transforms the opposite way, qbSchema -> JSONSchema (using Json.prettyPrint(Json.toJson(todoSchema))). 
  
How do I do vice versa (JSON Schema -> qbSchema)?

**Comment by koegel** [comment]
I will get back to you with an answer to your question but maybe JSONForms is more close to what you are looking for? http://github.eclipsesource.com/jsonforms

**Comment by slouc** [comment]
Yeah, I found another library (I used REST Assured's json-schema-validator module, https://github.com/jayway/rest-assured). I would be happier with using just one library though, so I thought if QB Project has this functionality it would be nice to have an example. Digging through the source, however, led me to a conclusion that this is not yet (fully) supported. Do you plan on adding it?

**Comment by koegel** [comment]
OK, that sounds good. No, I have to admit, currently there is no immediate plans to fully support this.

**Comment by edgarmueller** [comment]
As Max said there are no immediate plans of supporting JSON schema validation in qb, but we do want to support validation via a separate [JSON Schema validation library](https://github.com/eclipsesource/play-json-schema-validator) which qb will be based on in the future. Unfortunately, we can't exactly tell when we'll find time to do the rewrite.
